network/marks.csv

```diff
@@ -92,6 +92,7 @@ MCNL,CG,no,McNeill Hill,-39.444191609,176.696548844,366.277,NZGD2000,2008-05-26T
 METH,LI,no,Methven,-43.591354801,171.575302859,452.6573,NZGD2000,2010-09-10T00:00:00Z,9999-01-01T00:00:00Z
 MING,CG,no,Minginui,-38.61686,176.74973,458,WGS84,2010-04-28T00:00:00Z,9999-01-01T00:00:00Z
 MKNO,CG,no,Makino Road,-39.703375975,176.028832116,832.205,NZGD2000,2011-05-11T00:00:00Z,9999-01-01T00:00:00Z
+MKP1,XX,no,Makororipoint Landslide 1,-38.6635620167,178.105180756,67.305,WGS84,2025-03-17T20:32:38Z,9999-01-01T00:00:00Z
```

Comment:
Normally we don't put hours, minutes and seconds in the timestamp in this table.